pkg/runner/local_exec.go

```diff
@@ -81,6 +83,12 @@ func (*LocalExecutableRunner) Run(ctx context.Context, input *api.RunInput, ow i
 
 	testcase := plan.TestCases[seq]
 
+	// Create the assets directory
+	assetsDir := filepath.Join(input.EnvConfig.WorkDir(), input.TestPlan.Name, input.RunID, "assets")
```

Comment:
Same comment as above - `assetsDir` -> `hostAssetsDir` in case you agree that it is nice to be explicit that this is the host directory, just so that we are consistent with the other runner.

Reply:
We should also have one per test instance.

Reply:
@Stebalien ugh. I did that and forgot to push because I put it on a different branch... pushed!